client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -523,10 +542,10 @@ export default {
         block: tab.block,
         title: tab.title || blockToTab.title,
       }
-      this.updateTabs(newTab, tabIndex, block)
+      this.updateTab(newTab, tabIndex, block)
     },
 
-    updateTabs (newTab, tabIndex, block) {
+    updateTab (newTab, tabIndex, block) {
```

Comment:
And these makeTab, updateTab, createTab things. Don't see why they are necessary.